dcos-log/api/v2/routes.go

```diff
@@ -9,40 +9,49 @@ import (
 	"github.com/gorilla/mux"
 )
 
+const (
+	taskPath       = "/task/frameworks/{frameworkID}/executors/{executorID}/runs/{containerID}"
+	taskBrowsePath = taskPath + "/files/browse"
+	podPath        = taskPath + "/tasks/{taskPath}"
+	podBrowsePath  = podPath + "/files/browse"
+	discoverPath   = "/task/{taskID}"
+	componentPath  = "/component"
+)
+
 // InitRoutes inits the v1 logging routes
 func InitRoutes(v2 *mux.Router, cfg *config.Config, client *http.Client, nodeInfo nodeutil.NodeInfo) {
 	// browse sandbox files
 	wrappedBrowseFiles := middleware.Wrapped(http.HandlerFunc(browseFiles), cfg, client, nodeInfo)
-	v2.Path("/task/frameworks/{frameworkID}/executors/{executorID}/runs/{containerID}/files/browse").Handler(wrappedBrowseFiles).Methods("GET")
-	v2.Path("/task/frameworks/{frameworkID}/executors/{executorID}/runs/{containerID}/tasks/{taskPath}/files/browse").Handler(wrappedBrowseFiles).Methods("GET")
+	v2.Path(taskBrowsePath).Handler(wrappedBrowseFiles).Methods("GET")
+	v2.Path(podBrowsePath).Handler(wrappedBrowseFiles).Methods("GET")
 
 	// task logs
 	wrappedTaskLogHandler := middleware.Wrapped(http.HandlerFunc(filesAPIHandler), cfg, client, nodeInfo)
-	v2.Path("/task/frameworks/{frameworkID}/executors/{executorID}/runs/{containerID}/{file}").Handler(wrappedTaskLogHandler).Methods("GET")
-	v2.Path("/task/frameworks/{frameworkID}/executors/{executorID}/runs/{containerID}/tasks/{taskPath}/{file}").Handler(wrappedTaskLogHandler).Methods("GET")
+	v2.Path(taskPath + "/{file}").Handler(wrappedTaskLogHandler).Methods("GET")
```

Comment:
Here and elsewhere in `InitRoutes` we could use `path.Join` to be consistent, but it's no big deal.